examples/client/client.c

```diff
@@ -1221,66 +1221,70 @@ static const char* client_usage_msg[][78] = {
  || defined(HAVE_CERTIFICATE_STATUS_REQUEST_V2)
         "-W <num>    Use OCSP Stapling (1 v1, 2 v2, 3 v2 multi)\n",     /* 41 */
         "            With 'm' at end indicates MUST staple\n",          /* 42 */
+#if defined(WOLFSSL_TLS13) && defined(HAVE_CSR_TLS13MULTI)
```

Comment:
Does this support need gated? Should it be on by default for TLS v1.3 with OCSP?
Perhaps the name could be improved like `WOLFSSL_TLS_OCSP_MULTI`? 
If the macro is kept can you make sure it is available as an option in configure.ac and also documented?

Reply:
Agree. Default is multi OCSP stapling on TLS13. It is able to disable it by `--enable-ocspstapling=no-multi`

Reply:
Doc update 
[PR157](https://github.com/wolfSSL/documentation/pull/157)